### PR TITLE
Image URL's resolved to https

### LINKED ISSUE #20: Remove Logo svg file

### PR DESCRIPTION
**Pull request by hchawla1**
Fixes #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ If you are unfamiliar with IBM Bluemix and the Globalization Pipeline service ch
 
 You can create a new bundle right from the editor without having to use the Globalization Pipeline service's dashboard.
 
-![create bundle](images/create_bundle.png)
+![create bundle](https://github.com/IBM-Bluemix/gp-vscode-plugin/blob/master/images/create_bundle.png)
```
Comment (srl295) [inline README.md]: this shouldn't be needed, but makes some sense.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "gp-vscode-plugin",
     "displayName": "Globalization Pipeline Extension",
     "description": "Connect to the Globalization Pipeline service on Bluemix",
-    "icon": "images/logo.svg",
+    "icon": "images/logo.png",
```
Comment (srl295) [inline package.json]: does this need to have a full URL with https or is it OK?

File: README.md (hunk 1 above)
Comment (hchawla1) [inline README.md]: Image URLs in README.md and CHANGELOG.md need to resolve to https URLs.

Comment (hchawla1) [inline README.md]: and that's an image in our README

File: package.json (hunk 2 above)
Comment (hchawla1) [inline package.json]: I don't think so..Let me find out

**Pull request merged**